<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowClass</class>
 <widget class="QMainWindow" name="MainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>516</width>
    <height>344</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>电磁兼容综合电参量测试仪</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../data/SJzw3415.qrc">
    <normaloff>:/images/zw3415.png</normaloff>:/images/zw3415.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <property name="spacing">
     <number>10</number>
    </property>
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>连接设置</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <property name="topMargin">
        <number>4</number>
       </property>
       <property name="horizontalSpacing">
        <number>10</number>
       </property>
       <property name="verticalSpacing">
        <number>2</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>串口</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>波特率</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QComboBox" name="serialPort"/>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="baud">
         <property name="currentIndex">
          <number>3</number>
         </property>
         <item>
          <property name="text">
           <string>1200</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>4800</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>9600</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>19200</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>38400</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>57600</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>115200</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QSpinBox" name="slaveID">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>254</number>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>仪表地址</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QTableView" name="resultTable">
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
     </widget>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btFTest">
        <property name="text">
         <string>开始滤波测量</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btClear">
        <property name="text">
         <string>清除记录</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btExportFile">
        <property name="text">
         <string>输出到文件</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>516</width>
     <height>23</height>
    </rect>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="tearOffEnabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="actionAbout_SJzw3415"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionQuit">
   <property name="text">
    <string>退出</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionAbout_SJzw3415">
   <property name="text">
    <string>关于电参量综合测量系统</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../data/SJzw3415.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindowClass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>214</x>
     <y>236</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAbout_SJzw3415</sender>
   <signal>triggered()</signal>
   <receiver>MainWindowClass</receiver>
   <slot>aboutSoftware()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>257</x>
     <y>171</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btFTest</sender>
   <signal>toggled(bool)</signal>
   <receiver>MainWindowClass</receiver>
   <slot>ctrlMeterTimer(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>343</x>
     <y>149</y>
    </hint>
    <hint type="destinationlabel">
     <x>351</x>
     <y>112</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btClear</sender>
   <signal>clicked()</signal>
   <receiver>MainWindowClass</receiver>
   <slot>clearTable()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>405</x>
     <y>141</y>
    </hint>
    <hint type="destinationlabel">
     <x>400</x>
     <y>150</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btExportFile</sender>
   <signal>clicked()</signal>
   <receiver>MainWindowClass</receiver>
   <slot>exportToFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>455</x>
     <y>141</y>
    </hint>
    <hint type="destinationlabel">
     <x>445</x>
     <y>150</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>aboutSoftware()</slot>
  <slot>changePortInfo()</slot>
  <slot>ctrlMeterTimer(bool)</slot>
  <slot>clearTable()</slot>
  <slot>exportToFile()</slot>
 </slots>
</ui>
